bin/config.js

```diff
@@ -52,7 +52,7 @@ module.exports = {
     store: loadFactory(config.get('HARVEST_STORE_PROVIDER') || 'file', 'harvest.store')
   },
   aggregator: {
-    precedence: [['clearlydefined', 'licensee', 'scancode', 'fossology', 'cdsource']]
+    precedence: [['reuse', 'clearlydefined', 'licensee', 'scancode', 'fossology', 'cdsource']]
```

Comment:
Is there a reason to make reuse the highest priority tool? I'm wondering if we could make clearlydefined the highest precedence, followed by reuse.

Reply:
Oh, that's simply a left-over of several tests I made because I couldn't find a certain bug. Of course, I'm open to any kind of priority sequence - there's no real reason to make that number 1.